tracer_options.go

```diff
@@ -130,3 +130,9 @@ func (tracerOptions) Tag(key string, value interface{}) TracerOption {
 		tracer.tags = append(tracer.tags, Tag{key: key, value: value})
 	}
 }
+
+func (tracerOptions) BaggageRestrictionsConfig(cfg *BaggageRestrictionsConfig) TracerOption {
+	return func(tracer *Tracer) {
+		tracer.baggageRestrictionsConfig = cfg
```

Comment:
I wanted to copy: https://github.com/uber/jaeger-client-go/blob/master/tracer_options.go#L57 but what happens if metrics doesn't exist before that option is run?